feat: add ability to add text stroke

AioiLight opened the pull request:
![af8e1b7b-0c8a-4722-833c-df1b38b0df26](https://github.com/user-attachments/assets/af7d0862-a31e-43f9-9415-acab1ea98dd5)

This PR is adding ability to stroke to texts.

fixes: #578 

Added 2 properties (`WebkitTextStrokeWidth`, `WebkitTextStrokeColor`) and 1 shorthands (`WebkitTextStroke`). When stroke is enabled, `paint-order: stroke;` and `stroke-linejoin: round;` are automatically set to prevent the stroke from obscuring the text.

I don't have a deep understanding of all the code so further improvements may be needed.

shuding reviewed (commented):
Looks great, thank you! We need some tests though (check out /tests).

AioiLight commented:
Sorry for the sudden change.
Since `textStroke` is not a valid property for `ReactNode`, I changed it to `WebkitTextStroke`, which is a non-standard CSS property name.
I have also added some tests, but the paint-order on the test does not seem to work well it depending on the version of resvg.

shuding commented:
Thanks!
